docs/git_tips.md

```diff
@@ -31,7 +31,7 @@ The following resources can provide background on how Git works:
 
 ## Configuring the output of "git log"
 
-By default, the date that "git log" displays is the "author date." In Chromium,
+By default, the date that "git log" displays is the "author date".In Chromium,
```

Comment:
It is missing a space after the dot to completely fix the typo.

Reply:
oh yeah, I have corrected that. thanks.